conans/assets/templates/new_v2_autotools.py

```diff
@@ -0,0 +1,238 @@
+import textwrap
+
+from conans.assets.templates.new_v2_cmake import source_cpp, source_h, test_main
+
+conanfile_lib = textwrap.dedent("""
+    import os
+
+    from conan import ConanFile
+    from conan.tools.gnu import AutotoolsToolchain, Autotools
+    from conan.tools.layout import basic_layout
+    from conan.tools.files import chdir
+
+
+    class {package_name}Conan(ConanFile):
+        name = "{name}"
+        version = "{version}"
+
+        # Optional metadata
+        license = "<Put the package license here>"
+        author = "<Put your name here> <And your email here>"
+        url = "<Package recipe repository url here, for issues about the package>"
+        description = "<Description of {package_name} here>"
+        topics = ("<Put some tag here>", "<here>", "<and here>")
+
+        # Binary configuration
+        settings = "os", "compiler", "build_type", "arch"
+        options = {{"shared": [True, False], "fPIC": [True, False]}}
+        default_options = {{"shared": False, "fPIC": True}}
+
+        exports_sources = "configure.ac", "Makefile.am", "src/*"
+
+        def config_options(self):
+            if self.settings.os == "Windows":
+                del self.options.fPIC
+
+        def layout(self):
+            basic_layout(self)
+
+        def generate(self):
+            at_toolchain = AutotoolsToolchain(self)
+            at_toolchain.generate()
+
+        def build(self):
+            with chdir(self, self.source_folder):
+                self.run("autoreconf --force --install")
+            autotools = Autotools(self)
+            autotools.configure(self.source_folder)
+            autotools.make()
+
+        def package(self):
+            autotools = Autotools(self)
+            autotools.configure(self.source_folder)
```

Comment:
This extra ``configure()`` shouldn't be necessary here, the project has been configured and built before (and it can be assumed that it has been configured and built)

Reply:
Yes, I agree. I had to put it there because of the local flow, I was still checking how we can do so that the autotools sets the package_folder in a previous steps...